docs/debugging.md

```diff
@@ -96,7 +96,7 @@ Here's a sample of some websocket frames from the Debugger attaching:
 
 ### Communication
 
-Lastly, it's worth mentioning that Developer Tools is an *advanced* subject
+Lastly, it's worth mentioning that Developer Tools is an _advanced_ subject
```

Comment:
Does "is an advanced" make sense here since you're talking about "developer tools" wouldn't "are an advanced" make a little more sense? Maybe I'm reading it wrong here.

Reply:
haha